app/containers/Applications/ApplicationRowItemContainer.tsx

```diff
@@ -6,10 +6,10 @@ import Link from 'next/link';
 export const ApplicationRowItem = props => {
   const {ciipApplication = {}} = props;
   const statusBadgeColor = {
```

Comment:
This object should be defined outside of the component (at the top of this file is fine), and should probably have a `Record` type